lib/bundler/plumber/database.rb

```diff
@@ -100,7 +100,7 @@ def self.update!(options={})
         if File.directory?(USER_PATH)
           if File.directory?(File.join(USER_PATH, ".git"))
             Dir.chdir(USER_PATH) do
-              command = %w(git fetch --all)
+              system "git fetch --all"
```

Comment:
Ahh I see the problem, this code just resets hard to the same master you already have, since never fetches. The first command assignment is being overwritten.

I think the code should be:
```ruby
command = %w(git fetch --all)
command << %w(git reset --hard origin/master)
command << '--quiet' if options[:quiet]
```
What you think?